CMakeLists.txt

```diff
@@ -16,24 +16,25 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.2)
 SET(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/CMake)
 
 IF(CMAKE_GENERATOR MATCHES "^Visual Studio 14 2015" OR CMAKE_GENERATOR MATCHES "^Visual Studio 15 2017")
-  OPTION(USE_VCPKG_DEPS "Build vcpkg dependencies locally" OFF)
+  OPTION(USE_VCPKG_DEPS "Build vcpkg dependencies locally" ON)
 ENDIF()
 
 if(USE_VCPKG_DEPS)
+  # This must be before the project call for tool-chain to work correctly
   message("Note: Using locally built vcpkg dependencies.")
   include(VcpkgDeps)
 endif()
 
 PROJECT(CorsixTH_Top_Level)
 
-INCLUDE(CheckCXXCompilerFlag)
-CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
-CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
-IF(COMPILER_SUPPORTS_CXX11)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-ELSEIF(COMPILER_SUPPORTS_CXX0X)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
-ENDIF(COMPILER_SUPPORTS_CXX11)
+# Allow CMake to use folders in IDEs which support it
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+# C++14 Required
```

Comment:
I'm good with bumping the target language to C++14, but is it actually related to this commit in any way?

Such an important change should either go in it's own commit, or with the introduction of C++14 features into the code.